sdcm/cluster.py

```diff
@@ -4574,8 +4574,10 @@ def check_cluster_health(self):
             # Don't run health check in case parallel nemesis.
             # TODO: find how to recognize, that nemesis on the node is running
             if self.nemesis_count == 1:
-                for node in self.nodes:
-                    node.check_node_health()
+                with adaptive_timeout(Operations.HEALTHCHECK, node=self.nodes[0],
```

Comment:
does `adaptive_timeou`t support multiple nodes? This feels as a hack solution, we are waiting for a single node but actually we need all of the nodes